docs/coding-standards.md

```diff
@@ -47,10 +47,34 @@ To validate SASS files use `lando gulp validate:sass`.
 * Maximum allowed compound selectors is increased from 3 to 4 (selector-max-compound-selectors)
 * Alphabetical order of properties is not enforced (order/properties-alphabetical-order)
 
+**Rule:** `max-nesting-depth`
+* Recommend setting is `1` but then, what is the point of SASS
```

Comment:
Confused here. Above it says that there is a max-nesting-depth of four, but then here you say that recommended is one. Which is it?

Reply:
Updated comment.